script/chart-museum.sh

```diff
@@ -8,7 +8,7 @@ CHARTMUSEUM_PWD=${CHARTMUSEUM_PWD:-"password"}
 CHARTMUSEUM_NS=${CHARTMUSEUM_NS:-"chart-museum"}
 CHARTMUSEUM_VERSION=${CHARTMUSEUM_VERSION:-"3.9.0"}
 CHARTMUSEUM_HOSTNAME=${CHARTMUSEUM_HOSTNAME:-"chart-museum"}
-CHARTMUSEUM_IP=${DEX_IP}
+CHARTMUSEUM_IP=${LOAD_BALANCER_IP}
```

Comment:
```suggestion
CHARTMUSEUM_IP=${LOAD_BALANCER_IP:?"Not provided"}
```
I would add this check to make the script fail out loud when the env variable is not provided, a avoid some potential headaches 😅

Reply:
I did not add it here on purpose. Instead added the check below, inside function `pushChartToChartMuseum`, as it is the only operation that needs the IP.

Reply:
Yeah, I saw after adding my comment 🤐